### ISSUE TITLE
Better Textpack support

### ISSUE THREAD

**Issue by cara-tm** [textpattern/com_connect#68]
Please, add this first 2 lines into the English Texpack. Otherwise, the English translation is served as the current front-end language (in my case in French):

    #@public
    #Default language, en-us, en-gb

**Comment by philwareham** [comment]
Time for a new beta?

**Comment by Bloke** [comment]
Time for an actual release! :)

Though I'd like to fix #51, #52, #53, #66 and #67 first. I'll try and do that this weekend.

**Comment by Bloke** [comment]
Question is, do we release the 4.6.0 version or just merge the 4.7.0 to master and go with that?

**Comment by philwareham** [comment]
Is there any breaking changes in 4.70 branch (possibly the Textpacks? Are they compatible with Textpattern v4.6.x also?). If not then let's merge this into 4.6.0 (master) and keep the older version numbering IMO.

**Comment by Bloke** [comment]
Yeah, the Textpacks aren't strictly compatible with 4.6. As far as I'm aware (though I've not tried it) they choke on the comma-separated `@language` identifiers. Plus they use the old opinionated lang string designators, which is probably why we have a separate branch here.

We've also got global attributes in 4.7 to handle. I suspect that accounts for most of the issues I linked earlier and won't be backwards-compatible. So... **shrug**. Do we support 4.6 at all? Or just release the beta-3 as an official 4.6 version now and not support it further? Then merge 4.7 to master and carry on?
